project.py

```diff
@@ -13,41 +12,68 @@ def main():
             continue
         else:
             break
+    
+    while True:
+        try:
+            rows = int(input("How many rows do you want to have? "))

```

Comment:
```suggestion
            row_index = int(input("How many rows do you want to have? ")) - 1
```

Then you can use `board[row_index][column_index]` easily